src/types/resolveExpression.ts

```diff
@@ -591,7 +591,7 @@ function resolveStaticCall(
         const t = getExpType(ctx, e);
         if (!isAssignable(t, a.type)) {
             throwCompilationError(
-                `Invalid type "${printTypeRef(t)}" for argument ${idTextErr(a.name)}`,
+                `Cannot pass a value of type "${printTypeRef(t)}" to the parameter ${idTextErr(a.name)} of type "${printTypeRef(a.type)}"`,
```

Comment:
1. Let's move this into a helper function and reuse below.
2. It should say "expression" instead of "value"

Reply:
Have a look now please!